responsibleai/environments/responsibleai-vision-ubuntu20.04-py38-cpu/context/Dockerfile

```diff
@@ -20,10 +20,10 @@ RUN conda env create -p $AZUREML_CONDA_ENVIRONMENT_PATH -f conda_dependencies.ya
 RUN pip install --pre 'azure-ai-ml' 'azure-storage-blob<=12.13.0' 'numpy<1.24.0'
 
 # Install azureml packages
-RUN pip install 'azureml-dataset-runtime==1.48.0' \
-                'azureml-core==1.48.0' \
-                'azureml-mlflow==1.48.0' \
-                'azureml-automl-dnn-vision==1.48.0' \
+RUN pip install 'azureml-dataset-runtime' \
```

Comment:
do you want to resolve it to the latest available version instead of dropping the pin?

Reply:
I will set it explicitly to 1.49.0 because I don't think this logic works for Dockerfile, just conda_dependencies.yaml files:

https://github.com/Azure/azureml-assets/blob/main/scripts/azureml-assets/azureml/assets/environment/pin_package_versions.py#L19

Reply:
update: it seems it can't find the latest package version of automl-dnn-vision package, there is some error:

2023-02-27T21:42:12.4536065Z ##[debug]Looking up latest version of azureml-dataset-runtime
2023-02-27T21:42:12.5459201Z ##[debug]Latest version of azureml-dataset-runtime is 1.49.0
2023-02-27T21:42:12.5461662Z ##[debug]Looking up latest version of azureml-core
2023-02-27T21:42:12.5933374Z ##[debug]Latest version of azureml-core is 1.49.0
2023-02-27T21:42:12.5936807Z ##[debug]Looking up latest version of azureml-mlflow
2023-02-27T21:42:12.6121314Z ##[debug]Latest version of azureml-mlflow is 1.49.0
2023-02-27T21:42:12.6124209Z ##[debug]Looking up latest version of azureml-automl-dnn-vision
2023-02-27T21:42:12.6240223Z ##[debug]Latest version of azureml-automl-dnn-vision is None

I am trying to debug the reason why it is set to None for some reason

Reply:
It's not finding azureml-automl-dnn-vision because the GitHub agent on which the lookup is performed is running Python 3.11 and [azureml-automl-dnn-vision](https://pypi.org/project/azureml-automl-dnn-vision/) requires Python < 3.9.

Maybe it's best to limit the agent to Python 3.8 for now or find a good way to make the search Python version agnostic.

Reply:
Here's an approach that I believe will work: #413

Reply:
@imatiach-msft, would you please merge the main branch into yours? #413 was just merged.

Reply:
done! The package installation steps seem to be working now.